Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveHM_2nd.cpp

```diff
@@ -81,6 +81,9 @@ void FiniteDifferenceSolver::MacroscopicEvolveHMCartesian_2nd(
     amrex::GpuArray<int, 3> const& Mx_stag        = macroscopic_properties->Mx_IndexType;
     amrex::GpuArray<int, 3> const& My_stag        = macroscopic_properties->My_IndexType;
     amrex::GpuArray<int, 3> const& Mz_stag        = macroscopic_properties->Mz_IndexType;
+    amrex::GpuArray<int, 3> const& Bx_stag        = macroscopic_properties->Bx_IndexType;
+    amrex::GpuArray<int, 3> const& By_stag        = macroscopic_properties->By_IndexType;
+    amrex::GpuArray<int, 3> const& Bz_stag        = macroscopic_properties->Bz_IndexType;
```

Comment:
why is `B_stag` needed given that `M_stag` has already been read in?

Reply:
To keep the code generic, we go with separate staggerings for B, H, and M updates.